janusgraph-core/src/main/java/org/janusgraph/graphdb/configuration/JanusGraphConstants.java

```diff
@@ -42,6 +42,11 @@
      */
     public static final List<String> COMPATIBLE_VERSIONS;
 
+    /**
+     * Past versions of Titan Graph with which the runtime version shares a compatible storage format
+     */
+    public static final List<String> TITAN_BACKWARD_COMPATIBLE_VERSIONS;
```

Comment:
For conciseness and readability, I would just drop the `BACKWARD_` part — yes, we're only compatible with versions of Titan in the past, but it would read nicer (I think) if we just have `TITAN_COMPATIBLE_VERSIONS` and `JANUSGRAPH_COMPATIBLE_VERSIONS` and similarly `titan.compatible-versions` and `janusgraph.compatible-versions`.